articles/videos/get-started/01-architecture-your-tenant.md

```diff
@@ -0,0 +1,159 @@
+---
+description: What an Auth0 tenant is and how to configure it in the Auth0 Dashboard.
+classes: video-page
+---
+# Architecture: Your Tenant
```

Comment:
Small quibble, but (almost) all the other sections begin with a verb. This one begins with a nominalization. Aren't they actually "architecting" in this step when choosing how to use their tenants? So wouldn't "Architect" be the proper word?

Reply:
Yes - much better.